MediaPlayer/MediaControls.js

```diff
@@ -0,0 +1,150 @@
+		 * anything that is accepted by {@link agate/Icon.Icon}.
+		 *
+		 * @type {String}
+		 * @default 'menu'
+		 * @public
+		 */
+		menuIcon: PropTypes.string,
+
+		/**
+		 * A string which is sent to the `nextTrack` icon of the player controls. This can be
+		 * anything that is accepted by {@link agate/Icon.Icon}.
+		 *
+		 * @type {String}
+		 * @default 'nexttrack'
+		 * @public
+		 */
+		nextTrackIcon: PropTypes.string,
+
+		/**
+		 * `true` when the media is paused.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		paused: PropTypes.bool,
+
+		/**
+		 * A string which is sent to the `pause` icon of the player controls. This can be
+		 * anything that is accepted by [Icon]{@link agate/Icon.Icon}. This will be temporarily replaced by
+		 * the [playIcon]{@link agate/MediaPlayer.MediaControls.playIcon} when the
+		 * [paused]{@link agate/MediaPlayer.MediaControls.paused} boolean is `false`.
+		 *
+		 * @type {String}
+		 * @default 'pause'
+		 * @public
+		 */
+		pauseIcon: PropTypes.string,
+
+		/**
+		 * A string which is sent to the `play` icon of the player controls. This can be
+		 * anything that is accepted by {@link agate/Icon.Icon}. This will be temporarily replaced by
+		 * the [pauseIcon]{@link agate/MediaPlayer.MediaControls.pauseIcon} when the
+		 * [paused]{@link agate/MediaPlayer.MediaControls.paused} boolean is `true`.
+		 *
+		 * @type {String}
+		 * @default 'play'
+		 * @public
+		 */
+		playIcon: PropTypes.string,
+
+		/**
+		 * A string which is sent to the `previousTrack` icon of the player controls. This can be
+		 * anything that is accepted by {@link agate/Icon.Icon}.
+		 *
+		 * @type {String}
+		 * @default 'previoustrack'
+		 * @public
+		 */
+		previousTrackIcon: PropTypes.string,
+
+		/**
+		 * A string which is sent to the `repeat` icon of the player controls. This can be
+		 * anything that is accepted by {@link agate/Icon.Icon}.
+		 *
+		 * @type {String}
+		 * @default 'repeat'
+		 * @public
+		 */
+		repeatIcon: PropTypes.string,
+
+		/**
+		 * A string which is sent to the `shuffle` icon of the player controls. This can be
+		 * anything that is accepted by {@link agate/Icon.Icon}.
+		 *
+		 * @type {String}
+		 * @default 'shuffle'
+		 * @public
+		 */
+		shuffleIcon: PropTypes.string,
+
+		/**
+		 * The visibility of the component. When `false`, the component will be hidden.
+		 *
+		 * @type {Boolean}
+		 * @default true
+		 * @public
+		 */
+		visible: PropTypes.bool
+	},
+
+	defaultProps: {
+		menuIcon: 'menu',
+		nextTrackIcon: 'nexttrack',
+		pauseIcon: 'pause',
+		paused: true,
+		playIcon: 'play',
+		previousTrackIcon: 'previoustrack',
+		repeatIcon: 'repeat',
+		shuffleIcon: 'shuffle'
+	},
+
+	styles: {
+		css,
+		className: 'controlsFrame'
+	},
+
+	render: ({menuIcon, nextTrackIcon, pauseIcon, paused, playIcon, previousTrackIcon, repeatIcon, shuffleIcon, ...rest}) => {
+		delete rest.visible;
+
+		return (
+			<div className={css.mediaControls} {...rest}>
```

Comment:
I don't see any reason to define `mediaControls` here. Since we already have `controlsFrame`.
I think you can just remove className here and use `controlsFrame` class in less instead.